src/figuresClasses.ts

```diff
@@ -2,10 +1,111 @@
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  getArea(): number {
+    const aSide = this.a;
+    const bSide = this.b;
+    const cSide = this.c;
+
+    const semiP = (aSide + bSide + cSide) * 0.5;
 
-export class Triangle implements Figure {}
+    const area = Math.sqrt(
+      semiP * (semiP - aSide) * (semiP - bSide) * (semiP - cSide),
+    );
 
-export class Circle implements Figure {}
+    return Math.floor(area * 100) / 100;
+  }
 
-export class Rectangle implements Figure {}
+  constructor(
+    public color: string,
+    a: number,
+    b: number,
+    c: number,
+  ) {
+    const hasEnoughLength: boolean = a > 0 && b > 0 && c > 0;
+
+    const sum: number = [a, b, c].reduce(
+      (accum, current) => accum + current,
+      0,
+    );
+
+    const largestSide = Math.max(a, b, c);
+
+    const isTriangle: boolean = sum - largestSide > largestSide;
+
+    if (!hasEnoughLength) {
```

Comment:
[CODE STYLE] - When throwing an error, provide a detailed description of the error message as per checklist item 3. The current message 'some side is the wrong length' could be more descriptive, such as 'Triangle side lengths must be greater than zero'.